winui/Kanban/Workflows.md

```diff
@@ -0,0 +1,317 @@
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle10.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "WPF Issue";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Minimum and maximum properties are not working in dynamic update";
+            taskDetail.Category = "Review";
+            taskDetail.IndicatorColorKey = "High";
+            taskDetail.Tags = new List<string>() { "Bug Fixing" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle11.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "Kanban Feature";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Need to implement tooltip support for Kanban";
+            taskDetail.Category = "Review";
+            taskDetail.IndicatorColorKey = "High";
+            taskDetail.Tags = new List<string>() { "New Control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle12.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "New Feature";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Dragging events support for Kanban";
+            taskDetail.Category = "Closed";
+            taskDetail.IndicatorColorKey = "Normal";
+            taskDetail.Tags = new List<string>() { "New Control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle13.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "New Feature";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Swimlane support for Kanban";
+            taskDetail.Category = "Open";
+            taskDetail.IndicatorColorKey = "Low";
+            taskDetail.Tags = new List<string>() { "New Control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle14.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+            return taskDetails;
+        }
+        #endregion
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+By utilizing workflows, teams can effectively manage tasks and streamline the development process. As the project evolves, workflows can be adjusted to accommodate new requirements, making them a flexible and essential tool for project management.
```

Comment:
By following the code snippet, you will notice that the card picked from the `Open` state is not allowed to be dropped in the `Review` state because we have defined that the card can only move from the `Open` state to the `In Progress`, `Closed`, `Closed No Changes`, `Won't Fix` states and not to any other states.